src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -870,7 +872,19 @@ private void ScanSyntaxToken(ref TokenInfo info)
                         TextWindow.AdvanceChar();
                     }
 
-                    info.Text = TextWindow.GetText(intern: true);
+                    if (_errorCount++ > 100)
```

Comment:
@VSadov 

The native values hard coded an error limit.  IIRC it was 200 errors.  I think any minimum we choose should be at least that large.  

@sharwell 

What's the value in displaying 50,000 errors to the user vs. say 200?